### PR TITLE
Cordova feature, tested on ios and browser

### PR DESCRIPTION
**Pull request by RenaudRohlinger**
Ok so I achieved what we talked before without the need of a "www" folder. The only thing I added is a config.xml and updated main.js and index.html
The only problem is that we need to add cordova.js even in development mode so it will throw an error, but this is not really a problem, just a red line in the console. 
If the developer doesn't want to use cordova he should just remove the line : 
```<script type="text/javascript" src="cordova.js"></script>``` (as mentionned in the code)

I added your informations in the config.xml by default and updated the readme for the instructions.
Finally, I added myself as a contributor at the end of the readme if you ok with that ?

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -21,7 +21,9 @@
 <body>
 
     <div id="content"></div>
-
+    <!-- this file will throw an error because it doesn't exist yet, but it is necessary to let him empty, 
+    cordova will generate it in the future. If you don't need the cordova's features just remove it -->
+    <script type="text/javascript" src="cordova.js"></script>
```
Comment (leandr0ck) [inline src/index.html]: Maybe we could left this as a comment. I think the default is the web. What do you think?

File: README.md [hunk 2]
```diff
@@ -61,9 +61,42 @@ Run:
...
 ## Credits
 Big thanks to these great repos:
 
 https://github.com/belohlavek/phaser-es6-boilerplate
 
 https://github.com/cstuncsik/phaser-es6-demo
+
+## Contributors
+
+https://github.com/RenaudROHLINGER
```
Comment (leandr0ck) [inline README.md:102]: Dude, you are a great contributor!

File: src/index.html (hunk 1 above)
Comment (RenaudRohlinger) [reply src/index.html]: Oh yes, you are right ! And just replace the comment section to something like ```<!-- uncomment for cordova -->``` and update the readme in the cordova section
I can do this quickly if you want

Comment (leandr0ck) [reply src/index.html]: yes please, I think that should be ok

**Pull request merged**